include/iceflow/contentType.hpp

```diff
@@ -0,0 +1,10 @@
+#ifndef CONTENT_TYPE_HPP
+#define CONTENT_TYPE_HPP
+#include <cstdint>
+// ContentType header for defining custom TLV types
+
+namespace ContentType {
+const uint16_t JsonObject = 128; // nlohmann::json object
+}
+
+#endif // CONTENT_TYPE_HPP
```

Comment:
Please add trailing newlines to files

```suggestion
#endif // CONTENT_TYPE_HPP

```